src/standard-snippets.ts

```diff
@@ -0,0 +1,19 @@
+import {CompletionItemKind, InsertTextFormat} from 'vscode-languageserver';
+
+export default [
```

Comment:
Recommend giving this a name, as these are specifically snippets for when the user is in a JS file.